### PR TITLE
[FLINK-20163][docs-zh] Translate page "raw format" into Chinese

### PR DESCRIPTION
**Pull request by metaotao**
## What is the purpose of the change

Translate page "raw format" into Chinese.

 the page url is https://ci.apache.org/projects/flink/flink-docs-master/dev/table/connectors/formats/raw.html.

The doc located in "flink/docs/dev/table/connectors/formats/raw.zh.md".

## Brief change log

- translate "flink/docs/dev/table/connectors/formats/raw.zh.md".

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

**Comment by metaotao** [comment]
Hi, @wuchong ,Could you help review this PR in your free time, thanks~

**Comment by metaotao** [comment]
@flinkbot run azure

File: docs/dev/table/connectors/formats/raw.zh.md [hunk 1]
```diff
@@ -78,105 +78,105 @@ FROM(
 </div>
 </div>
 
-In contrast, you can also write a single column of STRING type into this Kafka topic as an anonymous string value in UTF-8 encoding.
+相反，你也可以将一个 STRING 类型的列作为 UTF-8 编码的匿名字符串值写入 Kafka topic。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/raw.zh.md:81]: 相对应的，你也可以将一个 STRING 类型的列以 UTF-8 编码的匿名字符串值写入 Kafka topic。

File: docs/dev/table/connectors/formats/raw.zh.md [hunk 2]
```diff
@@ -78,105 +78,105 @@ FROM(
...
     <tr>
       <td><h5>raw.endianness</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">big-endian</td>
       <td>String</td>
-      <td>Specify the endianness to encode the bytes of numeric value. Valid values are 'big-endian' and 'little-endian'.
-      See more details of <a href="https://en.wikipedia.org/wiki/Endianness">endianness</a>.</td>
+      <td>指定字节序来编码数字值的字节。有效值为'big-endian'和'little-endian'。
+      更多细节可查阅 <a href="https://en.wikipedia.org/wiki/Endianness">endianness</a>。</td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/raw.zh.md:117]:
```suggestion
      更多细节可查阅 <a href="https://zh.wikipedia.org/wiki/字节序">字节序</a>。</td>
```

**Comment by metaotao** [comment]
Hi @wuchong , Thanks for your suggestion, it has been modified according to your suggestion.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**